include/title_screen.h

```diff
@@ -0,0 +1,114 @@
+#ifndef GUARD_TITLE_H
+#define GUARD_TITLE_H
+
+#include "global.h"
+
+struct TitleStruct;
+
+typedef void (*TitleFunc)(struct TitleStruct*);
+
+struct TitleStruct {
+    u32 unk0;
+    u32 unk4;
+    u32 unk8;
+    u8 fillerC[4];
+    struct Unk10{
+        u32 unk0;
+        u8 filler4[4];
+        u32 unk8;
+        u16 unkC;
+        u8 fillerE[2];
+        u16 unk10;
+        u16 unk12;
+        u16 unk14;
+        u16 unk16;
+        u8 filler18[2];
+        u8 unk1A;
+        u8 unk1B;
+        u8 unk1C;
+        u8 filler1D[2];
+        u8 unk1F;
+        u8 filler20[6];
+        u8 unk26;
+        u8 unk27;
+    } unk10[7];
+    u8 unk128;
+    u8 filler129[3];
+    TitleFunc unk12C;
+    TitleFunc unk130;
+    s16 unk134;
+    u8 filler136[2];
+    u16 unk138;
+    u8 filler13A[4];
+    u16 unk13E;
+    u8 filler140[4];
+    TitleFunc unk144;
+};
+
+struct Unk_082D7850 {
+    u8 filler0[2];
+    u16 unk2;
+    u8 filler4[4];
+    void* unk8;
+    u8 fillerC[4];
+    u16* unk10;
+    u8 filler14[4];
+    void* unk18;
+};
+
+extern struct MusicPlayerInfo gUnk_030016A0;
```

Comment:
move this to gba/m4a_internal.h, and rename it to m4a.h (since we basically put everything there)